pkg/authorization/authorizer/scope/converter.go

```diff
@@ -214,8 +215,7 @@ func (e clusterRoleEvaluator) ResolveRules(scope, namespace string, clusterPolic
 		return []authorizationapi.PolicyRule{}, nil
 	}
 
-	ctx := kapi.WithNamespace(kapi.NewContext(), namespace)
```

Comment:
> how was this working before?

I think the getter against the cache ignored the field.  This is uncached until we get SharedInformers, so I don't have to plumb it through twice.